deploy/kubernetes/manifest.yaml

```diff
@@ -87,18 +43,11 @@ spec:
               mountPropagation: "Bidirectional"
             - name: plugin-dir
               mountPath: /csi
-            - name: device-dir
-              mountPath: /dev
         - name: csi-driver-registrar
-          image: quay.io/k8scsi/driver-registrar:v0.4.2
-          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.1.0
           args:
             - --csi-address=$(ADDRESS)
-            - --mode=node-register
-            - --driver-requires-attachment=true
-            - --pod-info-mount-version="v1"
             - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
-            - --v=5
```

Comment:
What's the default log level for csi-node-driver-registrar?

Reply:
not sure, let me add this back as it's useful.

Reply:
fixed